test/integration/cli-volumes.test.js

```diff
@@ -37,6 +41,8 @@ test('triton volume create ...', testOpts, function (tt) {
     tt.test('  cleanup leftover resources', function (t) {
         h.triton(['volume', 'delete', '-y', '-w', validVolumeName].join(' '),
             function onDelVolume(delVolErr, stdout, stderr) {
+                // If there was nothing to delete, this will fail so that's the
+                // normal case. Too bad we don't have a --force option.
```

Comment:
I created https://github.com/joyent/node-triton/issues/208 for the force option.